### DISCUSSION TITLE
CVE-2023-5129, CVE-2023-4863 (webp) vulnerability

### DISCUSSION THREAD

**Discussion by awais-ilyas** [SixLabors/ImageSharp#2556] [Q&A] [answered]
Hello Support Team,

In light of CVE-2023-4863/CVE-2023-5129 I would like to know if ImageSharp is vulnerable. If yes, what can we do to mitigate the problem?

https://nvd.nist.gov/vuln/detail/CVE-2023-5129
https://nvd.nist.gov/vuln/detail/CVE-2023-4863

**Comment by JimBobSquarePants** [comment, marked as answer]
Already answered in #2538 #2540 

Please ensure you take the time to read previous discussions. The links are on the first page of search results.
